api/api-doc/storage_service.json

```diff
@@ -1047,7 +1047,7 @@
          ]
       },
       {
-         "path":"/storage_service/cleanup_all",
+         "path":"/storage_service/cleanup_all/",
```

Comment:
Appending '/' may actually still be a breaking change. And it should be unnecessary.